### PR TITLE
feat(ui): intelligent adaptive access interface for MFA

### PR DESCRIPTION
**Pull request by remisultan**
## :id: Reference related issue. 

adaptive-access

https://github.com/gravitee-io/issues/issues/7689

## :pencil2: A description of the changes proposed in the pull request

This PR brings Adaptive Access new User Interface

## :memo: Test scenarios 

- Configure Adaptive Access and check if the behaviour is still consistent. 

⚠️ The Intelligent par can only be tested when the backend of Risk Assessment is merged

## :computer: Add screenshots for UI

Check the issue for UI screens

### CODE REVIEW

**Review by leleueri — CHANGES_REQUESTED** [review]
There is an error into the web console when the A-MFA with "Conditional' is selected

image

File: gravitee-am-ui/src/app/domain/applications/application/advanced/factors/mfa-activate/conditional/mfa-conditional.component.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {Component, EventEmitter, Input, Output} from "@angular/core";
+import {MatDialog, MatDialogRef} from "@angular/material/dialog";
+
+@Component({
+  selector: 'mfa-conditional',
+  templateUrl: './mfa-conditional.component.html',
+  styleUrls: ['./mfa-conditional.component.scss']
```
[the remark this answers is not in the document]
Comment (remisultan) [reply gravitee-am-ui/src/app/domain/applications/application/advanced/factors/mfa-activate/conditional/mfa-conditional.component.ts:22]: We keep so each component has it's associated scss file

File: gravitee-am-ui/src/app/domain/applications/application/advanced/factors/mfa-activate/mfa-activate.component.ts [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+      this.currentMode = MfaActivateComponent.modeOptions.CONDITIONAL;
+    } else if (this.enrollment.forceEnrollment) {
+      this.currentMode = MfaActivateComponent.modeOptions.REQUIRED;
+    } else {
+      this.currentMode = MfaActivateComponent.modeOptions.OPTIONAL;
+    }
+  }
+
+  modes() {
+    return MfaActivateComponent.modes.map(mode => MfaActivateComponent.modeOptions[mode]);
```
[the remark this answers is not in the document]
Comment (remisultan) [reply gravitee-am-ui/src/app/domain/applications/application/advanced/factors/mfa-activate/mfa-activate.component.ts:69]: I indeed did the change in sub component but not here.
Will change it

File: gravitee-am-ui/src/app/domain/applications/application/advanced/factors/mfa-activate/conditional/mfa-conditional.component.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (remisultan) [reply gravitee-am-ui/src/app/domain/applications/application/advanced/factors/mfa-activate/conditional/mfa-conditional.component.ts:22]: FP

**Comment by remisultan** [comment]
closed due to https://github.com/gravitee-io/gravitee-access-management/pull/1912

**Pull request closed without merging**